### ISSUE TITLE
Fen Error not playing anything through tmdb

### ISSUE THREAD

**Issue by sange9402** [Tikipeter/repository.tikipeter#706]
Hello,

I keep getting fen error- check log for more info.. when trying to play anything fen through tmdb... 

My Kodi event log just keeps saying "alarm clock" after trying and failing

???

Thanks!

**Comment by sange9402** [comment]
Hi,

Can you send me a link for that? Not sure I can find it (trying to look)
and I'm new here so don't understand how to find some of these things.


Thanks!

**Comment by Tikipeter** [comment]
https://github.com/Tikipeter/repository.tikipeter/issues/704#issuecomment-1062495488

**Comment by sange9402** [comment]
Got it, thanks! I'm not sure how or where to attach this file? Never gone
into files that way. Sorry I know this is a lot education to give.

**Comment by sange9402** [comment]
Hi, so I did the attachments and getting rid of the old fen player files in tmdb etc...it did not appear to work. Still getting the fen Error. Let me know what else we can do if anything! Thanks for the quick response

**Comment by xyzfre** [comment]
Did you try going into TMDB settings go to players and re do the players and when it asked to clear existing players put yes Hope it works out

**Comment by sange9402** [comment]
I went into tmdb helper's settings and didn't see that option you
mentioned....

I did this:

Downloaded an app like Xplore on your firestick.

Set it to show hidden files.

Then figure out how to navigate to the .kodi directory:

Android/data/org.xbmc.kodi/files/.kodi/userdata/addon_data/plugin.video.themoviedb.helper/players

Find the files and delete the existing ones.

Use Xplore to copy / extract the new files (from that GitHub zip but
extracted) to the above location.

Let me know if I missed something....
